solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -768,7 +768,7 @@ public void close() {
     } finally {
 
       sysPropsCacher.close();
-      customThreadPool.execute(() -> IOUtils.closeQuietly(cloudSolrClient));
```

Comment:
I think it was named well before.  The suffix should reflect the type -- a "SolrClient"